zip-reduce-shielded_tx_fee.rst

```diff
@@ -0,0 +1,135 @@
+
+
+Out of Scope for this Proposal
+============
+
+Discussion around transaction fees for T to T, T to Z or Z to T.
+
+
+Motivation
+============
+
+The default transaction fee presently is 0.0001 ZEC or 10,000 zats.
+With ZEC market price of $100, a user can send 10,000 transactions
+for 1 ZEC, that turns out to 1 cent per transaction and it rises
+with the increase in the price of ZEC.
+
+With increase in light wallet adoptions on mobile clients, many users
+will be new to the Zcash eco-system. And the fact that the
+transaction fees are paid by the sender (not the receiver) is
+new information to users who might use Zcash for e-commerce
+and app interactions that might result in several transactions each day.
+
+Privacy must not cost a premium. The cost of 10 shielded transactions
+buys 1GB of mobile data `in India today <https://www.cable.co.uk/mobiles/worldwide-data-pricing/>`.
+
+Zcash users must be able to do more with their ZEC balance
+than worry about paying the premium for shielded transactions.
+
+
+Requirements for consensus
+-------------------------
+
+The change to the conventional transaction fees must be undertaken sooner
+as it gets difficult to gain consensus with the growth in the network
+of wallets, exchanges, miners and third parties involved.
+
+The following parties need to be part of the consensus:
+
+* A guarantee from mining groups is required to include the lowered default fee transactions in the next block.
+* Wallet developers need to provide commitment to update the software to use the new fee.
+* Zcash documentation and community outreach must be undertaken to make the change known.
+
+
+Security and privacy considerations
+-----------
+
+Unique transaction fees can cause linkability within transactions,
+hence this change must be accepted by majority, if not all popular
+shielded transaction software providers before announcing the change.
+
+Varying/unique fees are bad for privacy, for the short term before blocks get full,
+it’s fine for everyone to use a constant fee. [#nathan-1]_
+
+Long term, the issue of fees needs to be re-visited as the blocks start getting consistently full.
+And the possibility of a DoS scenario becomes critical in nature. New ZIP with flexible fees [#ian-1]_
+along with scaling solutions need to be evaluated and applied.
+
+
+Activation
+============
+
+* The recommended fee of 0.00001 or 1000 zats must be activated at block 1046400 to coincide with the `Canopy upgrade. <https://z.cash/upgrade/canopy/>`
+* With a grace period of ~4 weeks (block 1080000) to upgrade to the reduced default transaction fee for zcashd and core clients used by exchanges & service providers.
+
+
+UX Guidance
+============
+
+Wallets must prevent users from altering the fee for shielded transactions.
+
+
+ZIP Editors
+-----------
+
+The current ZIP Editor is Aditya Bharadwaj, representing the Nighthawk Wallet.
+Additional Editors will be selected by consensus among the current Editors.
+
```

Comment:
If you accept the section I wrote about Denial of Service it might be nice to add:

```suggestion
Acknowledgements
~~~~~~~~~~~~~~~

Thanks to Nate Wilcox for improve the Denial of Service section.
```